.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,31 @@
+name: Site
+description: Report a platform or site problem - an issue that affects the rendering of the docs site
+labels: ['platform']
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for taking the time to submit this issue. Your input is really appreciated.
```

Comment:
I'd rather get straight to the point, I don't think this line is necessary or valuable to contributors.